### ISSUE TITLE
RDs as Industry Analysts

### ISSUE THREAD

**Issue by abrust** [msrd/Summit-2019-Unconference#23]
## TL;DR
Let’s discuss establishing an RD analyst registry, a clearinghouse for RDs who want to do industry analyst work and those who want to match other RDs with analyst opportunities.  RDs make great would-be analysts, given their combination of industry and technical knowledge, plus rock-star presentation skills and writing ability.  Doing this could help Microsoft, the RD Program and individual RDs.

## Longer version:
About 5 years ago, my career pivoted and I began to work as a legitimate industry analyst.  Over the years, I've worked to get other RDs involved.  **Today, there's ample precedent set for RDs to share their vast industry knowledge as industry analysts**, including:
- The RD Perspective Program, which I helped Patrick Hynds launch 
- A new venture from 1105 Media, called Redmond Intelligence, with which Vishwas Lele, Rocky Lhotka and Vinod Unny are also involved

But RDs could do more in this area, as most of you have the rather rare combination of corporate industry knowledge and technical knowledge, plus superior presentation skills and writing ability.  **Having RDs more visible as analysts could raise awareness of the RD Program, help Microsoft and help individual RDs and their companies, through a new type of exposure.**

I propose we discuss how to go about this, including the possibility of establishing an RD analyst registry, which could be a great clearinghouse for RDs who want to do analyst work -- reports, research notes, webinars, podcasts -- and RDs who want to match other RDs with analyst opportunities.

I'd like this to be a panel-led discussion amongst the whole group of us.  Patrick and Vishwas will be on the panel if this gets selected, and Rocky will be too, if we can get a slot that has us finishing by 2pm.  Other interested parties are welcome, and if this does get on the Unconference agenda, please do come prepared with ideas and questions.
